package.json

```diff
@@ -51,6 +51,7 @@
     "@govtechsg/tradetrust-utils": "^1.0.1",
     "@reduxjs/toolkit": "^1.6.1",
     "@types/gtag.js": "0.0.8",
+    "babel-plugin-transform-require-context": "^0.1.1",
```

Comment:
what is this for?

Reply:
there is a require.context in the code and in order to mock it in jest this is needed.

Reply:
unable to remove it, tried out what u suggested but still encountering the require.context() error in test.

Reply:
k